subcommands/keys/tuf_updates_rotate_offline_key.go

```diff
@@ -199,10 +199,14 @@ func doTufUpdatesRotateOfflineTargetsKey(cmd *cobra.Command) {
 func replaceOfflineRootKey(
 	root *client.AtsTufRoot, creds OfflineCreds, keyType TufKeyType,
 ) (*TufSigner, OfflineCreds) {
+	oldKey, err := FindOneTufSigner(root, creds, root.Signed.Roles["root"].KeyIDs)
```

Comment:
So, we are replacing the first found key?

Reply:
We are replacing one and the only found key.
In case if there are more than 1 matching keys - we error out.

Please, see the reference design for a reasoning why it is not secure to keep two or more actual signing keys in one file.

Reply:
>Please, see the reference design for a reasoning why it is not secure to keep two or more actual signing keys in one file.

That's not related to my question.

If we have more than one signing key (doesn't matter if they are in a file, files or DB) how we know that we are replacing the key that customers would like to rotate.

Reply:
I think `findTufSigners` searches through many files, so it can return many "keys/signers", and `FindOneTufSigner` just choose the first one.

Reply:
https://github.com/foundriesio/fioctl/blob/6a0a259b3d8010d0b5450a8020ab7719e292a7ed/subcommands/keys/tuf_utils.go#L225

Reply:
The `FindOneTufSigner` makes sure that `signers` list contains exactly one item before returning the first one.
Please, re-read its implementation.

Reply:
OK, so `creds` that contains a key file data identifies the key to be replaced.